docs/inference.rst

```diff
@@ -168,6 +168,19 @@ An example configuration file (named ``inference.ini``) is::
     ; polarization prior
     name = uniform_angle
 
+    ;
+    ;   Sampling transforms
+    ;
+    [sampling_parameters]
+    ; parameters on the left will be sampled in
+    ; parametes on the right
+    mass1, mass2 : mchirp, logitq
```

Comment:
Do we want to use `logitq` here? If we use `logitq` don't we have to add a sampling transform block for `q -> logitq` like : 
```
[sampling_transforms-logitq]
name = logit
inputvar = q
outputvar = logitq
```
Also, this documentation provides a basic example to run on GW150914, and the logit stuff is more like a fine tuning option in my opinion. So I'm not sure whether we should include that here at all. If we do include, then it might be nice to provide some documentation about what the logit transform is doing.